src/error.rs

```diff
@@ -87,9 +87,16 @@ pub(crate) enum Error<'src> {
   InitExists {
     justfile: PathBuf,
   },
+  IncludeRecursive {
+    cur_path: PathBuf,
+    recursively_included_path: PathBuf,
+  },
   Internal {
     message: String,
   },
+  InvalidInclude {
+    line_number: usize,
```

Comment:
We should include the path and the line number, since this error might originate from an include, and not the root justfile.